charts/prometheus-adapter/templates/cluster-role-binding-auth-delegator.yaml

```diff
@@ -2,11 +2,13 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
+  {{- if .Values.customAnnotations }}
+  annotations:
+  {{- toYaml .Values.customAnnotations | nindent 4 }}
+  {{- end }}
   labels:
     app: {{ template "k8s-prometheus-adapter.name" . }}
```

Comment:
this is redundant with: "app.kubernetes.io/part-of"

Reply:
done